tests/test_files.py

```diff
@@ -522,21 +522,6 @@ def pressurcooker_test_files():
     return download_fixture_files(PRESSURECOOKER_SUBS_FIXTURES)
 
 
-@pytest.fixture
-def youtube_test_file():
```

Comment:
I don't think I understand why this is no longer needed

Reply:
Because it was never used! See the deleted function argument below with the same name (pytest fixtures are referred to in function argument by their names). It was unused there, so I deleted it, and then it wasn't used anywhere else.